app/lucy/src/app/components/admin-tools/request-cell/request-cell.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { AccessRequest } from 'src/app/models/AccessRequest';
+import { UserService } from 'src/app/services/user.service';
+import { AccessRequestResponseModalEmitterResponse } from '../access-request-response-modal/access-request-response-modal.component';
+
+@Component({
+  selector: 'tr[app-request-cell]',
+  templateUrl: './request-cell.component.html',
+  styleUrls: ['./request-cell.component.css']
+})
+export class RequestCellComponent implements OnInit {
+
+  responding: boolean = false
+
+  get userUserame(): string {
```

Comment:
try to implement like return ((this.request || {}).requester || {} ).preferredUsername || ''